doc/context.md

```diff
@@ -0,0 +1,290 @@
+  include: document,
+  exclude: '.ad-banner'
+});
+```
+
+### Combine `exclude` with `include`
+
+To test specific sections of the page, while skipping parts within that section you can pass an object with an `include` and `exclude` property. `include` tells axe what elements to test, and `exclude` tells axe to skip certain included sections. The following shows how to test the `main` and `footer` elements in a page, except for any `.ad-banner` element:
+
+```js
+await axe.run({
+  include: ['main', 'footer'],
+  exclude: '.ad-banner'
+});
+```
+
+**Note**: When an element is both included and excluded, the selector that matches the nearest ancestor takes priority. I.e. if a node's the grandparent is included, but the parent is excluded, then the node is excluded. If the node's grandparent is excluded, but its parent is included, the node is included.
+
+## Select From Prior Tests
+
+It is possible to `include` or `exclude` nodes from prior tests. This lets you either skip those issues, or repeat a test while you are working on a solution. This can be done using the `target` property from an axe result:
+
+```js
+// Grab the target property from all color-contrast violations:
+const violation = priorResult.violations.find(
+  ({ id }) => id === 'color-contrast'
+);
+const targets = violation.nodes.map(issue => issue.target);
+// Run axe, only testing the prior color-contrast violations
+await axe.run(targets);
+
+// Or, to exclude those nodes from the test:
+await axe.run({ exclude: targets });
+```
+
+**Important**: It is not possible to pass a single `target` property to axe. This must be wrapped in an array. This is because axe-core's `target` property is itself an array. See [Implicit Frame And Shadow DOM Selection](#implicit-frame-and-shadow-dom-selection) for details.
+
+## Limit Frame Testing
+
+To include or exclude specific sections within a frame can be done with a `fromFrames` selector object. This property takes an array of selectors, the first to select the frame element(s) and the last to select the element(s) to include or exclude. The following shows how to test all `form` elements a `#paymentFrame` frame or iframe:
+
+```js
+// Test each <form> inside each #paymentFrame frame or iframe:
+axe.run({ fromFrames: ['#paymentFrame', 'form'] });
+```
+
+To select elements in nested frames, axe will need a selector for each level of nesting. Because axe tests frames recursively, this can be any number of levels deep. The following shows how to test the `form`, inside an `#inner`, inside an `#outer` iframe:
+
+```js
+// Test <form> inside, #inner, inside #outer:
+axe.run({ fromFrames: ['iframe#outer', 'iframe#inner', 'form'] });
+```
+
+The `fromFrames` object can be used as part of an `exclude` or `include` property. It can be by itself, or as part of an array along with other selectors. The following example shows how to exclude all `.ad-banner` elements on the top window, as well as any that are part of the first level of iframes:
```

Comment:
```suggestion
The `fromFrames` object can be used as part of an `exclude` or `include` property. It can be by itself or as part of an array along with other selectors. The following example shows how to exclude all `.ad-banner` elements on the top window, as well as any that are part of the first level of iframes:
```